vendor/wp-now/src/download.ts

```diff
@@ -13,7 +13,7 @@ import getWpNowPath from './get-wp-now-path';
 import { output } from './output';
 import { isValidWordPressVersion } from './wp-playground-wordpress';
 
-function httpsGet(url: string, callback: Function) {
+function httpsGet(url: string, callback: (res: IncomingMessage & FollowResponse) => void) {
```

Comment:
I also noticed this Typescript issue when checking the code. I opened the PR https://github.com/Automattic/studio/pull/427 to address it.